conf/pg_defaults.yml.dist

```diff
@@ -0,0 +1,18 @@
+# This file is loaded if PG is loaded without a webwork2 lib
+#
+# The following are configuration options that are needed by PG
+# and are a slimmed down list from those of WEBWORK_ROOT/conf/defaults.config
+
+options:
+  webworkDirs:
+    tmp: /tmp
+  externalPrograms:
```

Comment:
`externalPrograms` is originally from `site.conf` (`site.conf.dist`) if I am not mistaken.

Also - at present some command line options were being provided as part of the settings there, at least `--no-shell-escape` for various latex varieties. As those switched are important, they should be included here in some manner.

Reply:
@taniwallach the purpose of this is to allow PG to run without any of the webwork configuration files.  The external programs are needed, so they are put in this file.  

Note: if there is the presence of ww2, then the config files are loaded as usually.  

I'll look at the `--no-shell-escape` option.  For what I was trying to do, I do think they were needed.

Reply:
I think that @taniwallach is pointing out that your comment says this is a slimmed down list from `WEBWORK_ROOT/conf/defaults.config`, and yet the only thing here from that file is webworkDirs{tmp} and everything else is actually from `site.conf`.

There is a lot more from `defaults.conf` that will probably be needed.  Most likely many (if not all) of the `$pg{specialEnvironmentVars} will be needed.  One in particular is the `$pg{specialEnvironmentVars}{entryAssist}`.  For that see the discussion at https://github.com/drdrew42/renderer/pull/83#issuecomment-970584673.  This is where the server default setting should go.

Reply:
I agree that other fields will need to go into this PG environment.  I initially built this to get some testing running that was independent of webwork2.  I didn't test everything, but more importantly set up a structure to move forward.